src/styles/utils/variables.scss

```
@@ -0,0 +1,2 @@
+$calendar-day-size: 100px;
+$days-of-mouth: "tue" 1 1px, "wed" 2 2px, "thu" 3 3px, "fri" 4 4px, "sat" 5 5px, "sun" 6 6px;
```

```suggestion
$days-of-mouth: "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
```
u can use just index